scripts/tuple_tree_generator/tuple_tree_generator/generators/typescript.py

```diff
@@ -0,0 +1,191 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import re
+from collections import namedtuple
+from typing import Optional
+from jinja2 import Environment
+from jinja2.utils import Markup
+
+from .jinja_utils import loader
+
+from ..schema import Schema
+from ..schema import (
+    StructDefinition,
+    StructField,
+    SimpleStructField,
+    SequenceStructField,
+    ReferenceStructField,
+)
+
+int_re = re.compile(r"(u)?int(8|16|32|64)_t")
+CPP_TYPE = re.compile(r"^((?P<container>[^<>]+<(?P<contained>.*)>)|(?P<simple>[^<>]*))$")
+
+BUILTINS = ["string", "boolean", "bigint"]
```

Comment:
Uhm, `bigint` or switch to `std::string`?

Reply:
there are other fields that have a `uint64_t` so bigint needs to stay

Reply:
Mmmh, at this point let's stick with `uint64_t` and wait for people to complain that they are interpreted as float.

Reply:
I don't remember the exact reason for the switch (I think it had something to do with `bigint` or a wrapper being deserialized incorrectly), however this problem is not present anymore, so `ID`s are now `bigint`